### DISCUSSION TITLE
PCB V2

### DISCUSSION THREAD

**Discussion by gewin1988** [universam1/iSpindel#565] [Q&A]
Dear All!
First, thanks for all of your work in this project. Its outstanding!

I've downloaded a pcb for a diy ispindel months ago. I thought its good to order the pcb (takes months to arrive here from the supplier) and while im waiting, i can prepare for the DIY parts. I sadly noticed, that i downloaded a v2 pcb, and there is a v4 available now.
(Pictures attached). What do you think, can i make an "ispindel" from this pcb or i shall order a newer one? Is it worth to start the project with these? https://ibb.co/wNFGW1M
Thanks in advance!

**Comment by pppedrillo** [comment]
I can say V4 is proven to work. Don't have any experience with V2 tho.
